### PR TITLE
feature(FDS-383):[Dashboard]: Swap out to use the new Columns, Section, Stat components

### PR DESCRIPTION
**Pull request by rociobaena**
Swap out internal styles and components to use the new Columns, Section, Stat components internally from Cascara

### Snapshots
Stat was updating relying on the root class for `_` instead of `Stat` and adding a modifier class for clickable stats.

## PR Author Checklist

- [x] PR title adheres to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#specification)
- [x] I have finished reviewing the contents of this PR for accuracy
- [x] If needed, I have included a [Changeset](https://github.com/changesets/changesets) and it follows [Semantic Versioning principles](https://semver.org/)
- [x] I have completed all of the above and have enabled `auto-merge` for this PR

### CODE REVIEW

File: packages/cascara/src/components/Dashboard/Dashboard.module.scss [hunk 1]
```diff
@@ -4,25 +4,17 @@ $widget-background: #f0f0f0;
 
 .Dashboard {
   display: block;
-  columns: 30rem;
+  // columns: 30rem; //verify we dont need this
```
Comment (rociobaena) [inline packages/cascara/src/components/Dashboard/Dashboard.module.scss:7]: Removing this, the main layout in the dashboard is positioned in a single column:
image

File: packages/cascara/src/components/Dashboard/Dashboard.module.scss [hunk 2]
```diff
@@ -4,25 +4,17 @@ $widget-background: #f0f0f0;
...
   }
 
-  .Title {
-    display: block;
-    padding: 0.5em 0 0.25em;
-    margin: 0;
-  }
-
   .Data {
+    // we need this for Widget List
```
Comment (rociobaena) [inline packages/cascara/src/components/Dashboard/Dashboard.module.scss:17]: If we remove this, the rows lines disappear, that's why I keep them:
image

File: packages/cascara/src/components/Dashboard/widgets/WidgetStats.js [hunk 3]
```diff
@@ -18,10 +17,12 @@ const WidgetStats = ({ data, ...rest }) => {
   const dataState = getDataState(data);
 
   return (
-    <Widget {...rest} {...dataState} className={styles.Stats} height='auto'>
```
Comment (rociobaena) [inline packages/cascara/src/components/Dashboard/widgets/WidgetStats.js:21]: The Columns component works so fine here. Just the appearance was altered:
image
image
Should we keep `styles.Stats` or should we work on the class of the Stat component?

Comment (brianespinosa) [reply packages/cascara/src/components/Dashboard/widgets/WidgetStats.js:21]: We can take a look. If I recall, there is a Stat specific override that is making the grey background go away on these that we will want to keep. But we can also possibly simplify the styles.

**Comment by brianespinosa** [comment]
@rociobaena I am going to pull this branch today and take a look. Some of these I may need to poke around at to find out what is going on. Thank you for getting this started.

File: packages/cascara/src/components/Dashboard/Dashboard.module.scss (hunk 1 above)
Comment (rociobaena) [reply packages/cascara/src/components/Dashboard/Dashboard.module.scss]: Commented addressed pairing with @brianespinosa.

File: packages/cascara/src/components/Dashboard/Dashboard.module.scss (hunk 2 above)
Comment (rociobaena) [reply packages/cascara/src/components/Dashboard/Dashboard.module.scss]: This class is kept, no need to remove it.

**Review by lordpixel — APPROVED** [review]

**Pull request merged**